control-plane/roles/gardener/templates/shoot-dns-service/controller-deployment.yaml

```
@@ -10,10 +10,11 @@ providerConfig:
     image:
       repository: "{{ gardener_shoot_dns_service_image_name }}"
       tag: "{{ gardener_shoot_dns_service_image_tag }}"
+{% if gardener_shoot_dns_service_image_vector_overwrite %}
+    imageVectorOverwrite:
```

According https://github.com/gardener/gardener-extension-shoot-dns-service/blob/v1.46.0/charts/gardener-extension-shoot-dns-service/values.yaml#L52 it should be:
```suggestion
    imageVectorOverwrite: |
```